pkg/modulewriter/hcl_utils.go

```diff
@@ -71,6 +71,12 @@ func TokensForValue(val cty.Value) hclwrite.Tokens {
 	}
 
 	ty := val.Type()
+	if ty == cty.String {
+		s := val.AsString()
+		s = escapeBlueprintVariables(escapeLiteralVariables(s))
```

Comment:
Is this "call inside a call" necessary or is it just a way of converting 2 lines to 1 because most calls are "NO-OP"? Could it be done in the other order?

I personally prefer the 2-line approach but in any case, do not merge without a comment indicating that 2 things are happening here and indicate whether the order matters.